sound/soc/sof/ipc4-topology.h

```diff
@@ -55,6 +55,8 @@
 
 #define SOF_IPC4_INVALID_NODE_ID	0xffffffff
 
+#define INIT_PAYLOAD_WITH_OUTPUT_FMT	BIT(0)
```

Comment:
Please use enum if it is exclusive..

Reply:
The output format and base_config_extension is exclusive, but it is hard to say for future bits, let's stay with macro.

Reply:
> The output format and base_config_extension is exclusive, but it is hard to say for future bits, let's stay with macro.

Then we should use `if (process->init_payload_format & INIT_PAYLOAD_WITH_OUTPUT_FMT)` instead of `if (process->init_payload_format == INIT_PAYLOAD_WITH_OUTPUT_FMT)`

Reply:
@aiChaoSONG @ranj063 My point is that Using BIT(0) implies that other formats can coexist. For example, process->payload_with_output_fmt can be BIT(0) | BIT(1) | BIT(2). And we should copy the output format whenever the INIT_PAYLOAD_WITH_OUTPUT_FMT bit is set.

I don't know why we don't set output format if INIT_PAYLOAD_WITH_OUTPUT_FMT is set along with other bits.

> Ideally this token would contain only one of the bits set to identify the init format isnt it? Why not use enum or just macros for bit flags instead?

If `this token would contain only one of the bits set to identify the init format` is true, please use enum. I.e process->init_payload_format can be 0,1,2,3... I don't see any reason that we need to use a bit to represent a format.